docs/ThirdPartyHosting.md

```diff
@@ -10,41 +10,48 @@ Third-party hosts play an important role in the ecosystem since they provide an
-* No [Server Settings](./Walkthrough.md#server-settings) access
-
-The limitations happen for technical reasons, mostly because these features require a user to run a full node to use them.
-
-## Pros
+Table of contents:
+- [Advantages and Disadvantages](#advantages-and-disadvantages)
+   - [Pros](#pros)
+   - [Cons](#cons)
+- [Concerns For Use](#concerns-for-use)
+   - [Security](#security-concerns)
+   - [Privacy](#privacy-concerns)
+   - [Trust](#trust-concerns)
+- [Third Party Hosting FAQ](#third-party-hosting-faq)
+  - [Where is the list of BTCPay third-party hosts?](#where-is-the-list-of-btcpay-third-party-hosts)
+  - [How can one become a third-party host?](#how-can-one-become-a-third-party-host)
+  - [Are there any limitations in features when using a third-party host?](#are-there-any-limitations-in-features-when-using-a-third-party-host)
+  - [Can I enable the use of my Lightning Network node to others?](#can-i-enable-the-use-of-my-lightning-network-node-to-others)
+  - [What does the third-party host know about their users?](#what-does-the-third-party-host-know-about-their-users)
+
+## Advantages and disadvantages
+
+### Pros
 * Easy and quicker setup
 * Cheaper and in most cases free (depending if the host is premium or free)
 * Receive payments directly to your wallet
 * Private key never required (if it is, it's a scam!)
 
-## Cons
+### Cons
 * Security concerns
 * Privacy concerns
 * Limitation of features
 * No control over a server
 * Have to trust the owner of the server
 
-### Security concerns
+## Concerns For Use
+
+### Security Concerns
 [Trusted third parties are security holes](https://nakamotoinstitute.org/trusted-third-parties/#selection-7.6-6.2). By relying on someone else to manage a server for you, you are potentially exposing yourself to a certain attack vector.
 
-The most significant attack vector when using a third-party host is that a malicious and technically skilled host can create a forked version of BTCPay Server and modify it so that it either spy on your transactions or replace your [xpubkey](https://en.bitcoin.it/wiki/Deterministic_wallet_tools#Risks_of_Sharing_an_Extended_Public_Key_.28xpub.29) with his. This means that future payments made to you may end up in a malicious party wallet.
+The most significant attack vector when using a third-party host is that a malicious and technically skilled host can create a forked version of BTCPay Server and modify it so that it either spy on your transactions or replace your [xpubkey](https://en.bitcoin.it/wiki/Deterministic_wallet_tools#Risks_of_Sharing_an_Extended_Public_Key_.28xpub.29) with his. This means that future payments made to you may end up in a malicious party wallet. 
+
+It is impossible to know for certain, if the third party host was using a malicious fork. If you do not personally know the third party host (and even so) it is best to minimize trust in your third party host by doing the following:
```

Comment:
> If you do not personally know the third party host 

Perhaps just "if you don't trust"? Personally known seems a bit of an overstatement considering we live in a digital world :)